controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ArchiveAccessMaintainer.java

```diff
@@ -19,19 +22,32 @@
  */
 public class ArchiveAccessMaintainer extends ControllerMaintainer {
 
+    private static final String bucketCountMetricName = "archive.bucketCount";
+
     private final ArchiveBucketDb archiveBucketDb;
     private final ArchiveService archiveService;
     private final ZoneRegistry zoneRegistry;
+    private final Metric metric;
 
-    public ArchiveAccessMaintainer(Controller controller, Duration interval) {
+    public ArchiveAccessMaintainer(Controller controller, Metric metric, Duration interval) {
         super(controller, interval);
         this.archiveBucketDb = controller.archiveBucketDb();
         this.archiveService = controller.serviceRegistry().archiveService();
         this.zoneRegistry = controller().zoneRegistry();
+        this.metric = metric;
     }
 
     @Override
     protected boolean maintain() {
+
+        // Count buckets - so we can alert if we get close to the account limit of 1000
+        metric.add(bucketCountMetricName,
+                zoneRegistry.zones().all().ids().stream()
+                        .map(archiveBucketDb::buckets)
+                        .mapToLong(Collection::size)
+                        .sum(),
+                metric.createContext(Map.of()));
```

Comment:
We want at least the `system` dimension here. But ideally i think we should report per zone with `zone` dimension and we can aggregate in yamas for entire system if we want.